app/src/main/java/com/dkanada/gramophone/ui/fragments/player/card/CardPlayerFragment.java

```diff
@@ -286,8 +291,12 @@ public void onShow() {
 
     @Override
     public void onHide() {
+        shown = false;
```

Comment:
Where is this actually used?

Reply:
If it's not queried, it's probably a remaining thing from a previous approach

Reply:
It's not, removed